docs/en/getting-started/get-started-stack.asciidoc

```diff
@@ -0,0 +1,871 @@
++
+[source,ruby]
+----
+input {
+  beats {
+    port => 5044
+  }
+}
+
+# The filter part of this file is commented out to indicate that it
+# is optional.
+# filter {
+#
+# }
+
+output {
+  elasticsearch {
+    hosts => "localhost:9200"
+    manage_template => false
+    index => "%{[@metadata][beat]}-%{[@metadata][version]}-%{+YYYY.MM.dd}" <1>
+  }
+}
+----
++
+This configuration allows you to route events to {ls}, where you have full
+access the {ls} capabilities for collecting, enriching, and transforming
+data.
+
+
+[float]
+==== Start {ls}
+
+Use the command that works with your system. If you installed {ls} as a deb or
+rpm package, make sure the config file is in the `config` directory.
+
+*deb:*
+
+["source","sh",subs="attributes,callouts"]
+----------------------------------------------------------------------
+sudo /etc/init.d/logstash start
+----------------------------------------------------------------------
+
+*rpm:*
+
+["source","sh",subs="attributes,callouts"]
+----------------------------------------------------------------------
+sudo service logstash start
+----------------------------------------------------------------------
+
+*mac:*
+
+["source","sh",subs="attributes,callouts"]
+----------------------------------------------------------------------
+./bin/logstash -f logstash.conf
+----------------------------------------------------------------------
+
+*win:*
+
+["source","sh",subs="attributes,callouts"]
+----------------------------------------------------------------------
+bin\logstash.bat -f logstash.conf
+----------------------------------------------------------------------
+
+NOTE: The default configuration for {beats} and {ls} uses plain TCP. For
+encryption you must explicitly enable SSL when you configure {beats} and {ls}.
+
+{ls} reads from the {beats} input and indexes events into {es}. You haven't
+defined a filter section yet, so {ls} simply forwards events to {es} without
+additional processing. Next, you learn how to define the filter stage.
+
+[float]
+[[logstash-filter]]
+==== Define a filter to extract data from a field
+
+The system metrics collected by {metricbeat} include a field called `cmdline`
+that contains the full command-line argument used to start the process. For
+example:
+
+[source,json]
+----
+"cmdline": "/Applications/Firefox.app/Contents/MacOS/plugin-container.app/Contents/MacOS/plugin-container -childID 3
+-isForBrowser -boolPrefs 36:1|299:0| -stringPrefs 285:38;{b77ae304-9f53-a248-8bd4-a243dbf2cab1}| -schedulerPrefs
+0001,2 -greomni /Applications/Firefox.app/Contents/Resources/omni.ja -appomni
+/Applications/Firefox.app/Contents/Resources/browser/omni.ja -appdir
+/Applications/Firefox.app/Contents/Resources/browser -profile
+/Users/dedemorton/Library/Application Support/Firefox/Profiles/mftvzeod.default-1468353066634
+99468 gecko-crash-server-pipe.99468 org.mozilla.machname.1911848630 tab"
+----
+
+
+You might decide that you don't want to send the whole command-line argument to
```

Comment:
Alternate:

~~~
You might decide that you don't want to send the whole command-line argument to
{es}, and that sending the command's _path_ might be sufficient. One way to do that
would be by using a Grok filter to describe the pattern to extract, and to remove
the original field from the event to ensure it is not indexed. For example:
~~~